base/applications/network/dwnl/lang/it-IT.rc

```diff
@@ -0,0 +1,32 @@
+/*
+ * PROJECT:     ReactOS File Download Utility
+ * LICENSE:     See COPYING in the top level directory
+ * PURPOSE:     Italian resource file
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+ 
+LANGUAGE LANG_ITALIAN, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "Uso: dwnl URL [DESTINAZIONE]\n"
+    IDS_RESOLVING "Risolvere %s... "
+    IDS_DONE "fatto.\n"
+    IDS_LENGTH_FULL "Lunghezza: %I64u [%s]\n"
+    IDS_LENGTH "Lunghezza: %I64u\n"
+    IDS_FILE_SAVED "\nIl file è stato salvato.\n"
+    IDS_CONNECTING_TO_FULL "Connettersi a %s [%s]... "
+    IDS_CONNECTING_TO "Connettersi a %s... "
+    IDS_REDIRECTING_TO "Reindirizza a %s... "
+    IDS_SEND_REQUEST "Invio richiesta... "
+    IDS_BYTES_DOWNLOADED "%I64u byte di download"
```

Comment:
Better "scaricati" over "di download"

Reply:
The reason for "download" being there is because some software (in Italian localization of course) still keep the "download" terminology so I thought keeping such term for the sake of consistency among other English IT keywords in Italian.

But sure, I can change that to "scaricati" nevertheless.